packages/@aws-cdk/custom-resource-handlers/test/custom-resources-framework/expected/python-runtime/custom-resource-provider-core.ts

```diff
@@ -26,7 +26,7 @@ export class TestProvider extends CustomResourceProviderBase {
     super(scope, id, {
       ...props,
       "codeDirectory": path.join(__dirname, 'my-handler'),
-      "runtimeName": "nodejs18.x"
+      "runtimeName": "python3.10"
```

Comment:
node to python? Seems sketch.

Reply:
Oh wait, this is a test for python runtimes. How did we end up with node here? Ooooops.

Reply:
Yeah I also was confused but this is just GitHub showing the diff weird.

Reply:
I created new tests specific to python runtimes and node runtimes. I'm not sure why the diff is showing up like this.